spec/fixtures/conjur-ca.pem

```diff
@@ -0,0 +1 @@
+definitely a ca
```

Comment:
This intention has no newline to avoid having to add a newline in the assertions